src/google.rs

```diff
@@ -235,6 +235,7 @@ pub mod protobuf {
 
         use alloc::string::String;
         use alloc::vec::Vec;
+        #[cfg(feature = "borsh")]
```

Comment:
This isn't needed as the whole module depends on the `borsh` feature

Reply:
when using `parity-scale-codec` feature, it will report an error that borsh did not introduce.